pkgs/development/tools/turso-cli/default.nix

```
@@ -23,6 +26,13 @@ buildGo121Module rec {
     echo "v${version}" > internal/cmd/version.txt
   '';
 
+  postInstall = ''
+    installShellCompletion --cmd turso \
+      --bash <($out/bin/turso completions bash) \
+      --fish <($out/bin/turso completions fish) \
+      --zsh <($out/bin/turso completions zsh)
+  '';
+
   # Test_setDatabasesCache fails due to /homeless-shelter: read-only file system error.
   doCheck = false;
```

That error can be fixed like this

```suggestion
  preCheck = ''
    export HOME=$(mktemp -d)
  '';
```